functions/go/starlark/README.md

```diff
@@ -8,34 +8,46 @@ Run a Starlark script to mutate or validate resources.
 
 <!--mdtogo-->
 
-### FunctionConfig
+[Starlark] is a dialect of Python. It is commonly used as a configuration
+language. It is an untyped dynamic language with high-level data types,
+first-class functions with lexical scope, and garbage collection. You can find
+the spec for the Starlark language [here][spec]. You can also find its API
+reference [here][apiref].
 
 <!--mdtogo:Long-->
 
-[Starlark] is a dialect of Python. It is commonly used as a configuration language. It is an untyped dynamic language with
-high-level data types, first-class functions with lexical scope, and garbage
-collection.
+### Function
+
+[starlark] function contains an interpreter for the Starlark language. You can
+provide a Starlark script in the `functionConfig` to mutate or validate your
+resources. It is a light-weight way to write a KRM function. It is recommended
+to only build simple function with it. Generally, if you Starlark script is
```

Comment:
Just link to the book: https://kpt.dev/book/05-developing-functions/?id=approaches

This is an example where you don't want to repeat the book. If we have a python SDK in the future, we shouldn't need to change this doc.